.pre-commit-config.yaml

```diff
@@ -19,3 +19,14 @@ repos:
     rev: 0.27.0
     hooks:
       - id: check-github-workflows
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    - id: check-yaml
+    - id: check-toml
+    - id: check-merge-conflict
+    - id: end-of-file-fixer
+    - id: trailing-whitespace
+      exclude: .*\.md
+    - id: mixed-line-ending
+    - id: no-commit-to-branch
```

Comment:
I find this annoying locally there is no reason to have it beyond annoying users. 

I suggest removal in #6341